src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java

```diff
@@ -165,13 +161,12 @@ public void copiesFromCharArrayWithCharsetByNameToPath()
         final String input =
             "Hello, товарищ path #3 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```

Comment:
@a-korzhov here too

Reply:
> @a-korzhov here too

Guess I understood the logic, but then, why I dont have to put charset here, if the message is '... copied with UTF_8 charsets name'?

Reply:
@a-korzhov I meant "here too there is a problem". Just take care of not changing the test behaviour, only that it is correctly verifying the output content as per the `@todo` description

Reply:
@victornoel Done, check please.